generators/app/templates/src/app/index.ejs

```diff
@@ -7,9 +7,7 @@ import { apiSetup } from '@config/api';
 <%_ if(features.pushnotifications) { _%>
 import { configPushNotifications } from '@config/pushNotifications';
 <%_ } _%>
-<%_ if(features.login) { _%>
```

Comment:
`import './i18n';` at the top to make i18n work, i think its a bootstrap bug

Reply:
I'm not sure, we are not writing anything with i18n there. We have to make that import in every specific screen root component. But I have to check

Reply:
Yes! you were totally right!